homework-submission/week1/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoList from './TodoList.js';
+
+class App extends Component {
+  render() {
+    return <div className="App">
+        <header className="App-header">
+          <img src="https://russellinvestments.com/-/media/images/au/tiles/todolist.png" className="App-logo" alt="logo" />
+          <h1 className="App-title">Welcome to Todo List app</h1>
+        </header>
+        <p className="App-intro">This app is displaying three Todo items</p>
+        <TodoList />
+      </div>;
```

Comment:
I saw you are using this styling for more than a once which is fine. But I Highly recommend you to surround it with parentheses while every one else doing that not to be like them! but to make your code in other greater components more legible.
so you can just do it like so:
```javascript
class App extends Component {
  render() {
    return (
    <div className="App">
        <header className="App-header">
          <img src="https://russellinvestments.com/-/media/images/au/tiles/todolist.png" className="App-logo" alt="logo" />
          <h1 className="App-title">Welcome to Todo List app</h1>
        </header>
        <p className="App-intro">This app is displaying three Todo items</p>
        <TodoList />
      </div>
    );
  }
}
```

Reply:
Thanks a lot for your review and feedback I made the changes.